ballerina/Module.md

```diff
@@ -168,8 +168,9 @@ The `CREATE` statement is executed via the `execute` remote function of the clie
 
 ```ballerina
```

Comment:
Module.md and client.bal documentation is updated in sql module. Shall we do the same changes here too?

Reply:
Fixed in 43f2fe4